cernopendata/modules/fixtures/data/records/cms-tools-nanoaod-outreach-dimuon-spectrum.json

```diff
@@ -38,6 +38,11 @@
         "checksum": "adler32:c6ab092d",
         "size": 2415,
         "uri": "root://eospublic.cern.ch//eos/opendata/cms/software/DimuonSpectrumNanoAODOutreachAnalysis/dimuonSpectrum.py"
+      },
+      {
```

Comment:
Thanks, since this is a new file, we also have to update global `distribution` field, I'' do that while merging...

Reply:
Thanks!